Bio/Seq.py

```diff
@@ -770,8 +783,8 @@ def complement(self):
             ttable = _rna_complement_table
         else:
             ttable = _dna_complement_table
-        # Much faster on really long sequences than the previous loop based one.
-        # thx to Michael Palmer, University of Waterloo
+        # Much faster on really long sequences than the previous loop based
+        # one. Thx to Michael Palmer, University of Waterloo
```

Comment:
I'd have spelt out "Thanks" in full while changing the line wrapping, but its still fairly clear.